GTG/core/timer.py

```diff
@@ -82,3 +86,35 @@ def set_configuration(self, refresh_hour, refresh_min):
 
     def get_configuration(self):
         return self.config.get('hour'), self.config.get('min')
+
+    def time_parser(self, time):
```

Comment:
Please, provide docstring describing the function and also explaining what are arguments and return value of the function (don't forget to mention that ValueError is raised for invalid input)